README.md

```diff
@@ -1,13 +1,23 @@
-# Daml Platform - Observability Example
+# [Daml Enterprise](https://www.digitalasset.com/products/daml-enterprise) - Observability Example
 
-This repo demonstrates how you can set up your Daml deployment to leverage the observability
-features available, along with a few example Grafana dashboards for monitoring Daml services.
+This project demonstrates a [Daml
+Enterprise](https://www.digitalasset.com/products/daml-enterprise)
+deployment's observability features, along with example Grafana dashboards.  
+
+The project is self contained:  providing scripts to generate requests for
+which metrics are collected for display in the Grafana  dashboards.  
+
+This project is provided for illustrative purposes and local use only. The
+samples have been tested on MacOS and Linux -- some Docker images may not be
+available for Windows. This repository does not accept Pull Requests at the
```

Comment:
I'd suggest separating the statement on pull requests so that it is more obvious (i.e., to the last sentence in this section). You may want to emphasize it like [here](https://github.com/digital-asset/dev-env#template).

Reply:
I accepted your suggestion. Good to go.